connect/runtime/src/test/java/org/apache/kafka/connect/storage/KafkaConfigBackingStoreMockitoTest.java

```diff
@@ -1315,4 +1451,22 @@ private Map<String, Object> structToMap(Struct struct) {
         for (Field field : struct.schema().fields()) result.put(field.name(), struct.get(field));
         return result;
     }
+
+    // Manually insert a connector into config storage, updating the task configs, connector config, and root config
+    private void whiteBoxAddConnector(String connectorName, Map<String, String> connectorConfig, List<Map<String, String>> taskConfigs) {
```

Comment:
we don't use `whiteBox` now, so that can be simplified:
```java
    private void addConnector(String connectorName, Map<String, String> connectorConfig, List<Map<String, String>> taskConfigs) {
        for (int i = 0; i < taskConfigs.size(); i++)
            configStorage.taskConfigs.put(new ConnectorTaskId(connectorName, i), taskConfigs.get(i));
        configStorage.connectorConfigs.put(connectorName, connectorConfig);
        configStorage.connectorTaskCounts.put(connectorName, taskConfigs.size());
    }
```

Reply:
I would open a new pr for it.
Please check: https://github.com/apache/kafka/pull/16164, thanks!